app/models/agents/http_status_agent.rb

```diff
@@ -12,6 +12,7 @@ class HttpStatusAgent < Agent
 
     form_configurable :url
     form_configurable :disable_redirect_follow, type: :array, values: ['true', 'false']
+    form_configurable :changes_only, type: :array, values: ['true', 'false']
```

Comment:
I'll try it both ways.  If it carries values set as a dropdown true/false to boolean, I'll switch both.  I'll update here later.